src/test/java/gov/nih/nci/evs/api/controller/ObibSampleTest.java

```diff
@@ -114,4 +115,35 @@ public void testActive() throws Exception {
     assertThat(concept.getTerminology()).isEqualTo("obib");
     assertThat(concept.getActive()).isTrue();
   }
+
+  /**
+   * Test children not duplicated.
+   *
+   * @throws Exception the exception
+   */
+  @Test
+  public void testChildrenNotDuplicated() throws Exception {
```

Comment:
In the updates in SparqQueryCacheService, we have a child and parent part of the string. Is this testing that change? If so, we will want to make sure the parents aren't also duplicated, no?

Reply:
@bcarlsenca